README.md

```diff
@@ -50,9 +50,9 @@ git submodule update --init --recursive
 
 #### Install Dependencies
 
-On MacOS
+On macOS
```

Comment:
The subtitle is intended to be `MacOS`, following PyTorch readme: https://github.com/pytorch/pytorch#install-dependencies

Reply:
hmm.. seeing `macOS` as well https://github.com/pytorch/pytorch#install-pytorch

Reply:
Thanks @wenleix for reviewing, yes, the official name of the macOS has been `Mac OS X` then to `OS X` and now `macOS`, I saw the inconsistency in the readme thus change it .